packages/mergebot/src/compute-pr-actions.ts

```diff
@@ -114,9 +114,7 @@ function extendPrInfo(info: PrInfo): ExtendedPrInfo {
   const changereqReviews = info.reviews.filter((r) => r.type === "changereq") as ExtendedPrInfo["changereqReviews"];
   const staleReviews = info.reviews.filter((r) => r.type === "stale") as ExtendedPrInfo["staleReviews"];
   const hasChangereqs = changereqReviews.length > 0;
-  const approvedBy = getApprovedBy();
-  const pendingCriticalPackages = getPendingCriticalPackages();
-  const approverKind = getApproverKind();
```

Comment:
Dropped some of these to make them less of a hazard for code below; a lot of code was incorrectly reading out reviews for one package and applying them to the whole.